build_docs.sh

```diff
@@ -20,14 +22,17 @@ print_usage() {
 # Taken from SO post https://stackoverflow.com/a/246128
 script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-while getopts "r:e:yh" opt; do
+while getopts "r:e:iyh" opt; do
   case "${opt}" in
     r)
       opt_rosdistro=${OPTARG}
       ;;
     e)
       opt_repos=${OPTARG}
       ;;
+    i)
+      opt_generate_interfaces=1
```

Comment:
I think we can generate docs for interfaces by default.

I don't have a strong opinion about providing an option disable building interfaces docs. Is there a reason we might not want to?

Reply:
IMO, we should build interface docs by default and use the `-i` option to skip generation of interface docs.